server/commands.py

```diff
@@ -174,7 +174,7 @@ def ooc_cmd_pos(client, arg):
 def ooc_cmd_help(client, arg):
     if len(arg) != 0:
         raise ArgumentError('This command has no arguments.')
-    help_url = 'https://github.com/AttorneyOnlineVidya/tsuserver3'
+    help_url = 'https://github.com/Fronku/Attorney-Online-Vidya/blob/master/README.md'
```

Comment:
Don't link directly to the README, follow the old format.

Reply:
Alright.

Reply:
May I ask why, though?

Reply:
Consistency and less confusion when looking for other portions of the github. Not everyone is github literate.

Reply:
Done.